ui/raidboss/data/07-dt/trial/queen-eternal-ex.txt

```diff
@@ -0,0 +1,211 @@
+# -p A04B:1000.0 -ii A008 A009 A00D A00E A00F A010 A018 A026 A027 A02F A030 A053 A054 A056
+
+# Phase one pulled from -r XwHyk2NAKnbzZLf7 -rf 3
+# Phase two pulled from -r GBaArv7yzHnXck8D -rf 17
+# A lot of manual cleanup applied
+
+# Phase one
+0.0 "--sync--" InCombat { inGameCombat: "1" } window 0,1
+12.9 "--sync--" StartsUsing { id: "A00C", source: "Queen Eternal" }
+15.6 "Aethertithe" Ability { id: "A00C", source: "Queen Eternal" } duration 27.8
+27.4 "Retribute" Ability { id: "A011", source: "Queen Eternal" }
+28.3 "Retribute" #Ability { id: "A012", source: "Queen Eternal" }
+36.4 "Retribute" Ability { id: "A011", source: "Queen Eternal" }
+37.3 "Retribute" #Ability { id: "A012", source: "Queen Eternal" }
+45.4 "Retribute" Ability { id: "A011", source: "Queen Eternal" }
+46.3 "Retribute" #Ability { id: "A012", source: "Queen Eternal" }
+58.7 "Virtual Shift" Ability { id: "A019", source: "Queen Eternal" }
+
+# Wind miniphase
+67.8 "Laws of Wind" Ability { id: "A023", source: "Queen Eternal" }
+72.8 "Aeroquell" Ability { id: "A025", source: "Queen Eternal" }
+85.8 "Legitimate Force" Ability { id: ["A01E", "A020"], source: "Queen Eternal" }
+88.8 "Legitimate Force" Ability { id: ["A021", "A022"], source: "Queen Eternal" }
+91.9 "Wind of Change" #Ability { id: "A024", source: "Queen Eternal" }
+99.8 "World Shatter" Ability { id: "A01C", source: "Queen Eternal" }
+# End wind miniphase
+
+111.8 "Prosecution of War" Ability { id: "A00A", source: "Queen Eternal" }
+114.9 "Prosecution of War" Ability { id: "A00B", source: "Queen Eternal" }
+125.6 "Divide and Conquer x8" Ability { id: "A017", source: "Queen Eternal" } duration 7
+136.6 "Divide and Conquer" Ability { id: "7729", source: "Queen Eternal" }
```

Comment:
```suggestion
136.6 "Divide and Conquer (all)" Ability { id: "7729", source: "Queen Eternal" }
```

Reply:
I think `(all)` is potentially misleading here. This is the same lines firing in the same spot they fired originally, I'm just not sure how to convey it properly in a timeline. Maybe something like `(repeat)`...

Reply:
That's true, I was thinking of it from the perspective of normal mode. Maybe `(echoes)`, although that isn't clear either. Perhaps some of the other contributors have ideas?